src/coreclr/jit/emitxarch.cpp

```diff
@@ -5855,6 +5855,12 @@ bool emitter::IsMovInstruction(instruction ins)
         }
 
 #if defined(TARGET_AMD64)
+        case INS_movdqa32:
+        case INS_movdqa64:
+        case INS_movdqu8:
+        case INS_movdqu16:
+        case INS_movdqu32:
+        case INS_movdqu64:
```

Comment:
Why are these x64 only? They look to be valid on x86 as well based on the architecture manuals.

Reply:
Ah. should have been

```
        case INS_movdqa32:
        case INS_movdqa64:
        case INS_movdqu8:
        case INS_movdqu16:
        case INS_movdqu32:
        case INS_movdqu64:
#if defined(TARGET_AMD64)
        case INS_movq:
        case INS_movsxd:
#endif // TARGET_AMD64
        {
            return true;
        }
```